org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/XMLSchemaDiagnosticsTest.java

```diff
@@ -552,6 +552,36 @@ public void fuzzyElementNamesWithPrefixAndNoMatch() throws Exception {
 											ca(diagnostic, te(5, 12, 5, 17, "AElement2"), te(5, 28, 5, 33, "AElement2")));
 	}
 
+	@Test
+	public void elementThatCantHaveNodesHasNodes() throws Exception {
+		String xml =
+		"<?xml version=\"1.0\" encoding=\"UTF-8\"?>" + System.lineSeparator() +

```

Comment:
In our test we use System.lineSeperator() which depends on OS only if it is required. In this test, you can hard coded the line separator to have a simplier XML content.

More, we ends the line with // because when you format the test (Ctrl+Sift+F with Eclipse IDE), the format keeps the line.

In other words please replace  the line with 

`String xml = `<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n` + //`

and do like this for the other lines.